components/colorwheel/package.json

```
@@ -19,12 +19,13 @@
   },
   "peerDependencies": {
     "@spectrum-css/colorhandle": "^2.0.0",
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/tokens": "^7.0.0"
```

Let's update this one, too.

```suggestion
    "@spectrum-css/tokens": "^7.1.0"
```